docs/guides/1.18.0-upgrade-guide.md

```diff
@@ -14,6 +14,8 @@ The Terraform MongoDB Atlas Provider version 1.18.0 has a number of new and exci
 
 **Deprecations:**
 
+- `mongodbatlas_cluster` resource and corresponding data sources have been deprecated in favor of `mongodbatlas_advanced_cluster`. In order to learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide).
```

Comment:
Succint phrasing: "To learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide)."

Reply:
thanks, changed